# -*- conf-mode -*-
#
# NOTE: 2021-09-20 pip install cloudflare==2.8.15 fails with 
#  AttributeError: 'Distribution' object has no attribute 'convert_2to3_doctests'
# downgarding setuptools 58.0.4 to 53.0.0 fixes issue
#
#setuptools==53.0.0         # Require this first, to prevent later errors
#bs4			# 4.1.3 was installed
beautifulsoup4
celery==5.2.2
cloudflare==2.8.15
cryptography
Django>=3.2,<3.3
django-appconf		# 0.6 was installed
django-bootstrap5
django-csp==3.7
django-debug-toolbar>=1.7
django-environ
django-referrer-policy>=1.0
django-widget-tweaks>=1.3
elasticsearch>=7.0.0,<8.0.0
elasticsearch-dsl>=7.0.0,<8.0.0
factory-boy>=2.6.0
flower>=1.0.0
html5lib
jedi==0.17.2              # 0.18 has bug with ipython
lxml>=3.3.5
mysqlclient
mock>=2.0.0
mozilla-django-oidc==2.0.0
passlib
pyopenssl
pyquery
pytest
pytest-cov>=2.5.1
pytest-flakes
pytest-pycodestyle
pytest-django>=2.9.1
python-dateutil>=2.2
python-memcached>=1.48
scout-apm>=2.23.0
selenium
six
